src/main/java/org/folio/dew/service/BulkEditItemContentUpdateService.java

```diff
@@ -117,43 +116,40 @@ private ContentUpdateRecords applyContentUpdates(List<ItemFormat> itemFormats, C
         result.addToUpdated(updatedItemFormat);
         result.addToPreview(updatedItemFormat);
       } else {
-        if (isStatusUpdatingErrorNotExist(isStatusUpdatingExist, itemFormat, updatedItemFormat)) {
-          result.addToPreview(itemFormat);
-          var msg = NO_CHANGE_MESSAGE;
-          log.error(msg);
-          errorsService.saveErrorInCSV(jobCommand.getId().toString(), itemFormat.getId(), new BulkEditException(msg), FilenameUtils.getName(jobCommand.getJobParameters().getString(FILE_NAME)));
-        }
+        result.addToPreview(itemFormat);
+        var msg = NO_CHANGE_MESSAGE;
```

Comment:
The variable `msg` looks redundant. We can change `errorMessage.setValue(msg)` to `errorMessage.setValue(NO_CHANGE_MESSAGE)`.

Reply:
Fixed